src/LandingPage.js

```diff
@@ -0,0 +1,308 @@
+              infinite={true}
+              animationType="fadeout"
+              animationDuration={1000}
+              buttonsDisabled={true}
+              responsive={responsive}
+            >
+              <blockquote>
+                <span className="quote5 sidekick">
+                  Work hard in silence, let the success be your noise.
+                </span>
+                <cite className="author"> &mdash; Franck Ocean</cite>
+              </blockquote>
+              <blockquote>
+                <span className="quote1 sidekick">
+                  A goal without a plan is just a wish.
+                </span>
+                <cite className="author">
+                  {" "}
+                  &mdash; Antoine De Saint-Exupery
+                </cite>
+              </blockquote>
+              <blockquote>
+                <span className="quote2 sidekick">
+                  Master your weakness, grow your strenghts.
+                </span>
+                <cite className="author"> &mdash; Ryan Kahn</cite>
+              </blockquote>
+              <blockquote>
+                <span className="quote3 sidekick">
+                  Optimizing physical capacity requires training at
+                  unsustainable intensities.
+                </span>
+                <cite className="author"> &mdash; G.Glassman</cite>
+              </blockquote>
+              <blockquote>
+                <span className="quote4 sidekick">
+                  Fitness is a small, small world within the universe of
+                  movement.
+                </span>
+                <cite className="author"> &mdash; Ido Portal</cite>
+              </blockquote>
+              <blockquote>
+                <span className="quote5 sidekick">
+                  Work hard in silence, let the success be your noise.
+                </span>
+                <cite className="author"> &mdash; Franck Ocean</cite>
+              </blockquote>
+            </AliceCarousel>
+          </div>
+        </div>
+        <div className="section">
+          <div className="section-wrapper">
+            <div className="section-title">
+              READY TO TRACK <br></br> YOUR <br></br> PERFORMANCES?
+            </div>
+            <div className="section-text">
+              Recording your performances for each of your work out sessions is
+              the key of progress and consequently of success. #WORD helps you
+              to record, track and therefore have an accurate overview of your
+              progress. Enter your number of reps, the weights and all the
+              useful information to track to become a better athlete
+            </div>
+            <div className="section-buttons">
+              <button onClick={this.opensweetalert}>Sign up</button>
+              <button onClick={this.opensweetalert}>Log in</button>
+            </div>
+          </div>
+          <div className="section-carousel">
```

Comment:
> The carousel images to take the all height of the parent div. I'd like the images to take the all parent div height.

For that you, in this case, it's missing setting the height to the carousel and its direct child `div`.

```css
.alice-carousel,
.alice-carousel > div {
    height: 100%;
}
```